src/app/(dashboard)/attestation/utils/attestation-utils.ts

```diff
@@ -35,21 +37,29 @@ export type AttestationUuidDto = {
   attestationUuid: string;
 };
 
-export async function createAttestation({
+export async function createPrivateAttestation({
   address,
   privateData,
   signer,
+  chainId,
   salt,
 }: CreateAttestationBodyDto): Promise<AttestationUuidDto> {
-  // Sepolia private data schema
-  // @TODO Implement a mapping to reference the correct private data schema for each network
-  const schemaUID =
-    "0x20351f973fdec1478924c89dfa533d8f872defa108d9c3c6512267d7e7e5dbc2";
+  const chainConfig = chainsConfig[chainId];
+  if (!chainConfig) {
+    throw new Error(`No config found for chain id: ${chainId}`);
```

Comment:
oof, this is harsh, throwing an unhandled error on the frontend will make the user go to a blank application side error page. I think we should handle it diferently.

Specially on the case of chainID that the user can change at any moment on his wallet;

Maybe just return null?

same for the schemaUID below

Reply:
👍